### ISSUE TITLE
building failed from alpine image 

### ISSUE THREAD

**Issue by XGTIGER** [willhallonline/docker-ansible#41]
it is on  the master branch ,by docker-ansible/ansible211/alpine314,it seems caused by "cryptography",got:
```
File "/tmp/pip-build-env-p3651d2o/overlay/lib/python3.9/site-packages/setuptools_rust/setuptools_ext.py", line 103, in run
      build_rust.run()
    File "/tmp/pip-build-env-p3651d2o/overlay/lib/python3.9/site-packages/setuptools_rust/command.py", line 52, in run
      self.run_for_extension(ext)
    File "/tmp/pip-build-env-p3651d2o/overlay/lib/python3.9/site-packages/setuptools_rust/build.py", line 92, in run_for_extension
      dylib_paths = self.build_extension(ext)
    File "/tmp/pip-build-env-p3651d2o/overlay/lib/python3.9/site-packages/setuptools_rust/build.py", line 131, in build_extension
      metadata = json.loads(check_output(metadata_command))
    File "/usr/lib/python3.9/subprocess.py", line 424, in check_output
      return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
    File "/usr/lib/python3.9/subprocess.py", line 528, in run
      raise CalledProcessError(retcode, process.args,
  subprocess.CalledProcessError: Command '['cargo', 'metadata', '--manifest-path', 'src/rust/Cargo.toml', '--format-version', '1']' returned non-zero exit status 101.
  ----------------------------------------
  ERROR: Failed building wheel for cryptography
Failed to build cryptography
ERROR: Could not build wheels for cryptography which use PEP 517 and cannot be installed directly
The command '/bin/sh -c apk --no-cache add         sudo         python3        py3-pip         openssl         ca-certificates         sshpass         openssh-client         rsync         git &&     apk --no-cache add --virtual build-dependencies         python3-dev         libffi-dev         musl-dev         gcc         cargo         openssl-dev         libressl-dev         build-base &&     pip3 install --upgrade pip wheel &&     pip3 install --upgrade cryptography cffi &&     pip3 install ansible-core==2.11.3 &&     pip3 install mitogen ansible-lint jmespath &&     pip3 install --upgrade pywinrm &&     apk del build-dependencies &&     rm -rf /var/cache/apk/* &&     rm -rf /root/.cache/pip &&     rm -rf /root/.cargo' returned a non-zero code: 1
command failed with status: 1
```
thanks.

**Comment by willhallonline** [comment]
This should all be resolved now. At least it is showing all alpine builds correctly now and they are pushed to Docker Hub.

**Comment by willhallonline** [comment]
Resolved by v3.1.2
